integration_tests/src/test/scala/com/nvidia/spark/rapids/tests/tpch/TpchLikeSparkSuite.scala

```diff
@@ -91,113 +98,124 @@ class TpchLikeSparkSuite extends FunSuite with BeforeAndAfterAll {
     }).max())
   }
 
+  private def assertResult(expectedRowCount: Int)(df: DataFrame): Unit = {
```

Comment:
Can we rename this.  `assertResult` means that the two are equal and I got really confused when I initially scanned this patch.